tests/system/runtimes/test_nuclio.py

```diff
@@ -119,12 +122,85 @@ def test_serving_with_child_function(self):
 
         graph.to(
             ">>", "q1", path=f"v3io:///{self.stream_container}{self.stream_path}"
-        ).to(name="child", class_name="Identity", function="child")
+        ).to(name="child", class_name="Identity", function="child").to(
+            ">>",
+            "out",
+            path=f"/{self.stream_container}{self.stream_path_out}",
+            full_event=False,
+        )
 
-        function.add_child_function("child", child_code_path, "mlrun/mlrun")
+        graph.add_step(
+            name="otherchild", class_name="Augment", after="q1", function="otherchild"
+        )
+
+        graph["out"].after_step("otherchild")
+
+        function.add_child_function(
+            "child",
+            child_code_path,
+            image="mlrun/mlrun",
+        )
+        function.add_child_function(
+            "otherchild",
+            child_code_path,
+            image="mlrun/mlrun",
+        )
 
         self._logger.debug("Deploying nuclio function")
-        function.deploy()
+        url = function.deploy()
+
+        self._logger.debug("Triggering nuclio function")
+        resp = requests.post(url, json={"hello": "world"})
+        assert resp.status_code == 200
+
+        time.sleep(10)
```

Comment:
Why do we need such a long sleep here? Is it due to batch processing timeouts? Or something else?